viewer/package.json

```diff
@@ -6,12 +6,8 @@
     "url": "https://github.com/arkime/arkime.git"
   },
   "dependencies": {
-    "compression": "^1.6.2",
     "fs-ext": "^2.0.0",
-    "multer": "^1.4.5-lts.1",
-    "peek-stream": "^1.1.1",
-    "save-svg-as-png": "^1.4.5",
-    "through2": "^2.0.1"
+    "save-svg-as-png": "^1.4.5"
```

Comment:
You could put this in devdeps instead since we build this into the webpack bundle.